src/components/SkillCard.tsx

```diff
@@ -90,20 +93,19 @@ export const SkillCard = component$<Props>(({ skill, skillMatrix }) => {
 
 			<div class='w-full flex flex-col space-y-0'>
 				{skillMatrix.value
-					.sort((a, b) =>
-						Object.values(a)[0].skills[skill] < Object.values(b)[0].skills[skill]
-							? 1
-							: -1
-					)
+					.sort((s1, s2) => skillComparator(s1, s2, skill))
 					.slice(0, VISIBLE_SAILORS)
 					.map((skillMatrix, key) => {
-						const [name, { skills }] = Object.entries(skillMatrix)[0];
+						const [name, skillItem] = Object.entries(skillMatrix)[0];
 						return (
-							<div key={key} class='flex justify-between '>
-								<span class='text-sm font-normal'>{name}</span>
+							<div key={key} class='flex justify-between align-middle'>
+								<span class='text-sm font-normal flex flex-row gap-0.5'>
+									{skillItem.isCompany && getIcon('UserGroup')}
+									{name}
+								</span>
 								<SfRating
 									max={appStore.configuration.scoreRange.max}
-									value={skills[skill] || 0}
+									value={getSkillScore(skillItem, skill) || 0}
```

Comment:
Qui userei ?? piuttosto che ||
```suggestion
									value={getSkillScore(skillItem, skill) ?? 0}
```